### PR TITLE
DOCS-659: Improve robot page

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/manage/fleet/robots.md [hunk 1]
```diff
@@ -56,12 +52,23 @@ If you are managing a large fleet, you can use {{< glossary_tooltip term_id="fra
...
 
 ### History
 
 The configuration of your robot and the code it runs are kept separate to make debugging easier.
-The **HISTORY** tab shows a timestamped diff view of your robot's configuration changes.
+The **HISTORY** tab shows timestamped changes to your robot's configuration.
+
+If you want to revert changes that you made, you can load a previous configuration by clicking the **LOAD CONFIG** button next to the respective configuration.
+
+{{<gif webm_src="../../img/load-prev-config.webm" mp4_src="../../img/load-prev-config.mp4" alt="Load a previous config from the UI">}}
```
Comment (sguequierre) [inline docs/manage/fleet/robots.md:69]: Unfortunately this gif looks super blurry when 
I load it on the staging link-- maybe try reuploading?

File: docs/manage/fleet/robots.md [hunk 2]
```diff
@@ -22,16 +22,12 @@ Click the name of a robot to go to that robot's page, where you'll find a variet
...
-
-![The robot page for an example robot with the parts drop down open.](../../img/app-usage/part-drop-down.png)
-To delete a part or make it the main part, use the buttons in the top right of the **CONFIG** tab.
-
-![The CONFIG tab of a robot's page noting the location of the Make main part and Delete Part buttons.](../../img/app-usage/part-mgmt.png)
+The banner at the top of the robot page displays the robot's location, name, and a drop down list of all {{< glossary_tooltip term_id="part" text="parts" >}} of that robot.
 
 If you've connected your robot to a machine running `viam-server`, the banner also displays when the robot was last online, which version of `viam-server` it is running, the host name, the IP address or addresses, and its operating system.
 
+??
```
Comment (sguequierre) [inline docs/manage/fleet/robots.md:29]:
```suggestion
```

**Review by sguequierre — CHANGES_REQUESTED** [review]

**Review by sguequierre — APPROVED** [review]
lgtm but you probably want to change capitalization on the tabs/sub-tabs (**CONFIG** -> **config** etc.) before merging!

File: docs/manage/configuration.md [hunk 3]
```diff
@@ -253,7 +253,7 @@ To add a fragment to a robot:
 
 ![The fragments subtab](../img/fragments-tab.png)
 
-The components included in the fragment appear inside a read-only fragment section in the **Components** sub-tab.
+The components and services included in the fragment appear inside a read-only fragment section in the **Components** and **SERVICES** sub-tab.
```
Comment (npentrel) [inline docs/manage/configuration.md]:
```suggestion
The components and services included in the fragment appear inside a read-only fragment section in the **Components** and **Services** sub-tab.

```

File: docs/manage/parts-and-remotes.md [hunk 4]
```diff
@@ -42,6 +42,10 @@ Use the parts drop-down menu in the top banner of your robot's page on the [Viam
 
 ![The Viam app interface with the part drop-down open. A new part called "my-sub-part" is being created.](../img/parts-and-remotes/sub-part-config.png)
 
+To delete a sub-part or make it the main part, use the buttons in the top right of the **CONFIG** tab.
```
Comment (npentrel) [inline docs/manage/parts-and-remotes.md]:
```suggestion
To delete a sub-part or make it the main part, use the buttons in the top right of the **config** tab.
```

File: docs/manage/parts-and-remotes.md [hunk 5]
```diff
@@ -42,6 +42,10 @@ Use the parts drop-down menu in the top banner of your robot's page on the [Viam
 
 ![The Viam app interface with the part drop-down open. A new part called "my-sub-part" is being created.](../img/parts-and-remotes/sub-part-config.png)
 
+To delete a sub-part or make it the main part, use the buttons in the top right of the **CONFIG** tab.
+
+![The CONFIG tab of a robot's page noting the location of the Make main part and Delete Part buttons.](../img/parts-and-remotes/part-mgmt.png)
```
Comment (npentrel) [inline docs/manage/parts-and-remotes.md]:
```suggestion
![The config tab of a robot's page noting the location of the Make main part and Delete Part buttons.](../img/parts-and-remotes/part-mgmt.png)
```

File: docs/manage/configuration.md [hunk 6]
```diff
@@ -253,7 +253,8 @@ To add a fragment to a robot:
 
 ![The fragments subtab](../img/fragments-tab.png)
 
-The components included in the fragment appear inside a read-only fragment section in the **Components** sub-tab.
+The components and services included in the fragment appear inside a read-only fragment section in the **Components** and **Services** sub-tab.
+
```
Comment (npentrel) [inline docs/manage/configuration.md:257]:
```suggestion
```

**Pull request merged**